pages/api/check-results.ts

```diff
@@ -0,0 +1,40 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import { COOKIE_CHECK_ID_NAME } from '../../constants';
+import getOnfido from '../../helpers/onfido';
+import type { CheckResults } from '../../types/CheckResults';
+import { CheckResultsStatus } from '../../types/CheckResults';
+
+const endpointName = 'check-results';
+
+const onfido = getOnfido();
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse<CheckResults>) {
+  console.log(`[${endpointName}] Request received`);
+
+  const checkId = req.cookies[COOKIE_CHECK_ID_NAME];
+  const check = await onfido.check.find(checkId);
+
+  if (!checkId || !check) {
+    res.status(404).json({ isClear: null, status: CheckResultsStatus.notFound });
```

Comment:
To avoid https://en.wikipedia.org/wiki/Magic_number_(programming), I'm a fan of pulling these out like https://github.com/NEARFoundation/tx-tracking-app/blob/7ba69e69b6d90fbf999854b9c04e377c2dc21979/shared/helpers/statusCodes.ts

Reply:
Fixed and linting rule added!